Scripts/UI/Title/TitleScreen.cs

```diff
@@ -0,0 +1,42 @@
+using Godot;
+using System;
+
+public partial class TitleScreen : Control
+{
+	private Button startButton;
+	private Button exitButton;
+	private AudioStreamPlayer musicPlayer;
+
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready()
+	{
+		 // Initialize nodes
+        startButton = GetNode<Button>("VBoxContainer/NewGameButton");
+        exitButton = GetNode<Button>("VBoxContainer/ExitButton");
+        musicPlayer = GetNode<AudioStreamPlayer>("AudioStreamPlayer");
+
+        // Play background music
+        musicPlayer.Play();
+        // Connect button signals
+        startButton.Connect("pressed", new Callable(this, nameof(OnNewGamePressed)));
+        exitButton.Connect("pressed", new Callable(this, nameof(OnExitPressed)));
+	}
+
+	private void OnNewGamePressed()
+    {
```

Comment:
One more indent here would make this uniform with the rest of the class, which would be nice (:

Reply:
Removed, good call

Reply:
Updated indenting... or did ? XD